### PR TITLE
Milestone 3: Created Responsive Layout for Desktop Version of Portfolio

### PR DESCRIPTION
**Pull request by Kwentsir**
In this milestone we:

- created responsive layouts for the desktop version of the portfolio
- fixed errors and cleaned the code.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -372,3 +394,195 @@ footer p {
   position: absolute;
   left: 85%;
 }
+
+#contact-img2 #contact-img3 #contact-img4 {
+  display: none;
+}
+
+@media only screen and (min-width: 1080px) {
+  .background-image {
```
Comment (codepantha) [inline style.css:404]: - [ ] At the specified width, the navbar and the `intro `heading overlap as you can see below 👇. Please try to fix it in order to make your app look better. I suggest using margins 😎 
image

File: index.html [hunk 2]
```diff
@@ -104,11 +115,25 @@ <h2 class="recent-work__items"> Multi-Post Stories Gain+Glory </h2>
             </ul>
             <button type="button" class="see-projects">See Projects</button>
          </article>
+         <article class="article-card">
+            <img src="./images/Imgplaholder.svg" alt="Image placeholder">
+            <h2 class="recent-work__items"> Multi-Post Stories Gain+Glory </h2>
```
Comment (codepantha) [inline index.html:123]: - [ ] Please use the correct `font-family` as specified in the Figma design for your project titles. Do this for each of your `recent-work__items` classes. 🙏

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @Kwentsir 👋, 

# STATUS: CHANGES REQUIRED ♻️ 

Good job so far 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (sja-thedude) [reply index.html:123]: Good 👍 implementing the changes

File: style.css (hunk 1 above)
Comment (sja-thedude) [reply style.css:404]: - [ ] Kindly make sure to align the text in the centre as per the Figma design.
- Your design:
Screenshot 2022-03-26 at 10 01 51 PM

- Figma design:
Screenshot 2022-03-26 at 10 01 27 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Kwentsir, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job implementing the changes.

 ## Required
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -8,23 +8,34 @@
...
-          <ul>
+          <ul class="mobile">
               <li><a href="#" class="welcome"> Welcome </a></li>
-              <li> <a href="#"> <img src="./images/Union.svg" alt=""/></a></li>
+              <li> <a href="#" class="union"> <img src="./images/Union.svg" alt=""/></a></li>
+          </ul>
+          <ul class="desktop">
+              <li><a href="#" class="desktop-menu"> Portfolio </a></li>
+              <li><a href="#" class="desktop-menu"> About </a></li>
+              <li><a href="#" class="desktop-menu"> Contact </a></li>
```
Comment (AnselemOdims) [inline index.html:28]: - [ ] As already pointed out by the previous reviewer, please make sure the headline text is centered to match the original design

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Kwentsir , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by sinansevgi — COMMENTED** [review]
Hello @Kwentsir  👋,

_This is an additional review as a result of the second opinion request._

When you use percentage units for vertical properties, its value will be relative to the parent element’s height. For your project, this means margin-top for your page headline will be relative to the screen height of the device and will look different on different devices. Using percentages can be helpful if you are working with horizontal properties (width, left, right). But you should avoid using percentages on vertical properties (height, top, bottom) to prevent this kind of issue. 

You should use other units (rem, em, or calc function) to give margin-top to your headline in order to make your page look similar on all screen resolutions. 

Since this issue is relevant to your screen resolution and hard to identify by yourself, I think your project should get approved. 

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!
Sinan Sevgi

**Pull request merged**